### PR TITLE
fix: playwright trace is missing on ci

### PR DESCRIPTION
**Pull request by jerensl**
**Notes for Reviewers**

- Fix trace
- Remove firefox test to avoid data race

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/playwright.config.js [hunk 1]
```diff
@@ -38,7 +41,7 @@ module.exports = defineConfig({
       mode: 'on-first-retry',
     },
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
```
Comment (ianrwhitney) [inline ui/playwright.config.js:41]: curious as the test suite grows if we will have to consider only keeping traces if the fail for ci perfermance. 

 This option keeps the trace on retry even if it succeeds/fails. Is that correct?

**Review by ianrwhitney — COMMENTED** [review]
LGTM. one thought about future considerations. thanks

File: ui/playwright.config.js (hunk 1 above)
Comment (ianrwhitney) [reply ui/playwright.config.js:41]: curious thoughts on retain-on-failure option. As if we implemented 1 retry to reduce noise of flaky tests, we won't have to update this again. let me know your thoughts. Either way im good as along as there is awerness. thanks 🚀

**Review by ianrwhitney — APPROVED** [review]
LGTM. thanks 🚀 . 

Curious where the traces will be located and how PR owers will be able to view them? Any updates in docs required for contributers

**Review by leecalcote — APPROVED** [review]
Thank you, gents!

**Pull request merged**